src/components/cylc/commandMenu/Menu.vue

```diff
@@ -192,6 +198,10 @@ export default {
       return this.mutations
     },
 
+    title () {
+      return this.node.tokens.clone({ user: undefined }).id
```

Comment:
Sneaky, make sure this is nailed down with a test.

Reply:
Added a unit test